internal/cmd/plugin/command_install.go

```diff
@@ -48,7 +48,8 @@ func (c *command) install(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	dir, err := os.MkdirTemp("~/.confluent", "confluent-plugin-install")
+	confluentDir := filepath.Join(os.Getenv("HOME"), ".confluent")
+	dir, err := os.MkdirTemp(confluentDir, "confluent-plugin-install")
```

Comment:
`~/.confluent/plugin-install` is less redundant than `~/.confluent/confluent-plugin-install`.
```suggestion
	dir, err := os.MkdirTemp(confluentDir, "plugin-install")
```